imports/lib/models/base.ts

```diff
@@ -98,28 +97,28 @@ class Base<T extends BaseType> extends Mongo.Collection<T> {
 
   find(selector: FindSelector<T> = {}, options: FindOptions = {}) {
     return super.find(
-      _.extend({ deleted: false }, this[formatQuery](selector)),
+      { deleted: false as any, ...this[formatQuery](selector) },
```

Comment:
Do you intend for the `deleted: false` to be overrideable at the query site here, or do you mean for users to have to use the appropriate `findDeleted` or `findAllowingDeleted` accessor?

Because with object spreads, the last-mentioned value wins, which means that you can just override it from the query.

Reply:
Yeah it does seem like the behavior is questionable, but `_.extend` has the same behavior so this isn't a behavior change. I guess I can fix